Content.Client/GameTicking/Managers/ClientGameTicker.cs

```diff
@@ -19,18 +21,21 @@ namespace Content.Client.GameTicking.Managers
     public sealed class ClientGameTicker : SharedGameTicker
     {
         [Dependency] private readonly IStateManager _stateManager = default!;
-        [Dependency] private readonly IEntityManager _entityManager = default!;
+        [Dependency] private static readonly IEntityManager _entityManager = default!;
         [Dependency] private readonly IConfigurationManager _configManager = default!;
         [Dependency] private readonly SharedAudioSystem _audio = default!;
 
         [ViewVariables] private bool _initialized;
-        private Dictionary<NetEntity, Dictionary<string, uint?>>  _jobsAvailable = new();
+        private Dictionary<NetEntity, Dictionary<string, uint?>> _jobsAvailable = new();
         private Dictionary<NetEntity, string> _stationNames = new();
 
+        public static ClientGameTicker_RoundEndData_Container _roundEndContainer = new();
+
         /// <summary>
         /// The current round-end window. Could be used to support re-opening the window after closing it.
         /// </summary>
-        private RoundEndSummaryWindow? _window;
+        private static RoundEndSummaryWindow? _window;
+        private EscapeUIController _escapeController = new();
```

Comment:
As above, clientgameticker should just have an event as relevant rather than calling out of sim.

Reply:
the escapeUiController object here was so that I could access that "escape menu", this is done so that we can show and hide the button for the round end summary, at will. 
Ideally, it would show up in that menu, only after the round ends, and be hidden from the player when the round starts.
Without it, the button shows permanently, but only works at round end.
It's more of a user experience addon I wanted to do, but can remove if needed?